Gemfile.lock

```diff
@@ -526,4 +526,4 @@ DEPENDENCIES
   will_paginate
 
 BUNDLED WITH
-   1.14.6
```

Comment:
Since I was working on my other computer, [I had to use a gem `rb-readline` gem to get `byebug` to work](https://github.com/deivid-rodriguez/byebug/issues/118) (in hindsight I should have just done `gem install rb-readline).  I have since removed the gem but my bundler updated this line.  I dont think it's a big deal...